packages/blade/src/components/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,39 @@
+import type { ReactElement } from 'react';
```

Comment:
Thinking `components/IconButton`. But we can consider something else for internal components too.

Reply:
We should extract out common components into a separate directory and just keep the components we expose inside `components/*`. This could be similar to how we have the `utils` folder. Maybe have a `common/` that can include `IconButton`, `Label`, `ValidationState` etc. ?

Reply:
We also have other internal components which reside in the same `components/` directory like BaseButton, BaseInput, etc. 🤔

Reply:
Would rather suggest to keep it closer where it belongs so for Label we have a Form directory where we have kept it and all other utils and helpers for form also go there so here i think we can move it to `components/Button`. thoughts?

Reply:
Sounds good, it's anyway a button variation. Will move it

Reply:
Moved in https://github.com/razorpay/blade/pull/613/commits/963d8b8d376877a21c3cc3b37953cf486ad341bb